tests/shell/shell_recorder/tutorial_wrapper/markdown_shell_recorder.sh

```diff
@@ -210,3 +224,18 @@ done <<<"$BLOCKS"
 
 translate
 
+MOUNTPOINTS=$(kdb mount)
+
+if [ "$MOUNTPOINTS_BACKUP" != "$MOUNTPOINT" ];
+then
+IFS='
+'
+	TOUMOUNT=$(diff <(echo "$MOUNTPOINTS_BACKUP") <(echo "$MOUNTPOINTS") | grep -Eo "^>.*")
+	for line in $TOUMOUNT;
+	do
+		mp=$(sed -n 's/\(.*\)with name \(.*\)/\2/p' <<< "$line")
+		kdb umount "$mp"
```

Comment:
That sounds like a good idea. However, since this is not my code I will leave this job to anyone else who is interested in it 😊.